src/scripts/farming/Farming.ts

```diff
@@ -1220,6 +1220,21 @@ class Farming implements Feature {
             undefined,
             ['Detective Pikachu']
         );
+
+        this.berryData[BerryType.Hopo] = new Berry(
+            BerryType.Hopo,
+            [10800, 21600, 43200, 86400, 604800],
+            1,
+            0,
+            15000,
+            25,
+            [15, 40, 35, 30, 25],
+            49.3,
+            BerryColor.Gold,
+            5.8,
+            BerryFirmness.Very_Soft,
+            ['A truly mythical Berry. It is said to have been first created by a god.']
```

Comment:
```suggestion
            ['A hardy Berry native to harsh northern lands. If a wild Pokémon eats one of these Berries, that Pokémon's reactions will be dulled.']
```
Most of our berry descriptions use official Tag information, even if they don't make sense to pokeclicker mechanics. I've added one sentence from PLA and one generic descriptor for flavor text. We should also avoid theistic references lest we offend someone or are accused of blasphemy.

Reply:
Altered to make the reference to Arceus more clear